### PR TITLE
Carrier metadata update for UA - 380 (uk)

### PR DESCRIPTION
**Pull request by burunduk**

### CODE REVIEW

File: resources/carrier/uk/380.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# limitations under the License.
+
+# Data taken from: https://www.itu.int/oth/T02020000DB/en
+# Translation taken from https://uk.wikipedia.org/wiki/%D0%9A%D0%BE%D0%B4%D0%B8_%D0%BC%D0%BE%D0%B1%D1%96%D0%BB%D1%8C%D0%BD%D0%B8%D1%85_%D0%BE%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%BE%D1%80%D1%96%D0%B2_%D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%B8
+
+# Telesystems of Ukraine is commonly known as PEOPLEnet.
+
+38039|Голден Телеком
+38050|Vodafone Україна
+38063|lifecell
```
Comment (penmetsaa) [inline resources/carrier/uk/380.txt]: Looks like lifecell in Ukrainian is Лайфселл?
https://uk.wikipedia.org/wiki/Lifecell
If this is good can we update all lifecell entries?

File: resources/carrier/uk/380.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+38066|Vodafone Україна
+38067|Київстар
+38068|Київстар
+38073|lifecell
+38091|ТриМоб
+38092|PEOPLEnet
```
Comment (penmetsaa) [inline resources/carrier/uk/380.txt]: Similar case here. Is it "ПіплНет"?
https://uk.wikipedia.org/wiki/PEOPLEnet

File: resources/carrier/uk/380.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2013 The Libphonenumber Authors
+
```
Comment (penmetsaa) [inline resources/carrier/uk/380.txt]: Please change the year to 2018

File: resources/carrier/uk/380.txt (hunk 2 above)
Comment (burunduk) [reply resources/carrier/uk/380.txt]: There is no translation for this brand.
The wiki is the pronunciation. On company websites, I do not see the information that translates the brand.

**Comment by burunduk** [comment]
@AbdulmajidAlnouri, I copied this file from en/380.txt and translated the lines according to the information on the company's websites.

File: resources/carrier/uk/380.txt (hunk 1 above)
Comment (penmetsaa) [reply resources/carrier/uk/380.txt]: What about this one?

Comment (burunduk) [reply resources/carrier/uk/380.txt]: Similar to PEOPLEnet. I did not find use in the company of the translated brand.

**Comment by AbdulmajidAlnouri** [comment]
@burunduk Sorry I must been blind or something because I thought there was a discrepancy between the two. LGTM.

**Comment by penmetsaa** [comment]
@burunduk,
We will merge this PR after the present release (tentatively by end of today).
You may need to sync your branch with new master after our confirmation. Thanks for your patience.

**Comment by penmetsaa** [comment]
Hi,

Thanks for updating the branch with master one. Merged it.

**Pull request merged**